assets/src/dashboard/app/views/storyAnimTool/components.js

```diff
@@ -44,7 +44,7 @@ export const ActiveCard = styled.div(
       .map(
         (elementId) => `
           [data-element-id='${elementId}'] {
-            border: 2px solid ${theme.DEPRECATED_THEME.colors.bluePrimary};
```

Comment:
These changes are to non-user-facing UIs, so I just chose the most similar colors from the current theme.

Reply:
sounds good, we're gonna gut the story animation tool this sprint.